### PR TITLE
wisps no longer fall in chasms

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

prevents wisps from falling in chasms

## Why It's Good For The Game

they are flying orbiting magicy wisps, they shouldnt disappear if you fly over chasms with them

## Changelog
:cl:
fix: wisps no longer fall in chasms
/:cl:

### CODE REVIEW

File: code/datums/components/chasm.dm [hunk 1]
```diff
@@ -23,7 +23,8 @@
 		/obj/effect/collapse,
 		/obj/effect/particle_effect/ion_trails,
 		/obj/effect/dummy/phased_mob,
-		/obj/effect/mapping_helpers
+		/obj/effect/mapping_helpers,
+		/obj/effect/wisp
```
Comment (coiax) [inline code/datums/components/chasm.dm:27]:
```suggestion
		/obj/effect/wisp,
```

Trailing comma on the last one, keeps the diffs lighter.

**Review by coiax — COMMENTED** [review]
Out of scope:

Why wouldn't we just disable `/obj/effect` from falling into chasms entirely? What effects would we WANT to fall into chasms?

**Comment by Fikou** [comment]
> Out of scope:
> 
> Why wouldn't we just disable `/obj/effect` from falling into chasms entirely? What effects would we WANT to fall into chasms?

i dont know but im scared

File: code/datums/components/chasm.dm (hunk 1 above)
Comment (Mothblocks) [reply code/datums/components/chasm.dm]: we need this on contribution guidelines if like 3 of us are enforcing it

**Comment by Mothblocks** [comment]
> What effects would we WANT to fall into chasms?

Decals, I guess? When they were still effects and not just blended on?

**Pull request merged**